### PR TITLE
Feature/getall users

### PR DESCRIPTION
**Pull request by damola-sd**
# Description
GetAllUsers endpoint: Fixed all bugs and tested features

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: __test__/users.test.js [hunk 1]
```diff
@@ -123,31 +123,37 @@ describe('USER PASSWORD RESET', () => {
...
+      username: 'john'
+    };
+    const jwtToken = await jwt.generateToken(user);
     return request(server)
-      .get('/api/user/all')
+      .get('/api/user/john/users')
+      .set({ token: jwtToken })
       .then(res => {
-        expect(res.status).toBe(500);
+        expect(res.status).toBe(201);
```
Comment (iamcristos) [inline __test__/users.test.js]: status code 201 is when you created a new Item...
kindly use 200

File: api/controllers/mentor.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ module.exports = {
         include: [
           {
             model: models.Users,
+            attributes: {},
```
Comment (iamcristos) [inline api/controllers/mentor.js]: is there a reason you passing attributes as an empty object????

Comment (damola-sd) [reply api/controllers/mentor.js]: File still in development, Kindly ignore

File: __test__/users.test.js (hunk 1 above)
Comment (damola-sd) [reply __test__/users.test.js]: Changed the status codes now

**Pull request merged**